flexmeasures/data/services/scheduling.py

```diff
@@ -91,14 +93,30 @@ def load_custom_scheduler(scheduler_specs: dict) -> type:
     return scheduler_class
 
 
+def get_asset_or_sensor(id: int, entity_type):
```

Comment:
We should note the reason why we currently cannot simplify this by just passing around the class instead of the class name: i.e. the function arguments need to be serializable as job parameters.

Can't we write class methods (`.toJSON()`?) that serialize a `Sensor` as `{"sensor": <sensor_id>}` and an `Asset` as `{"asset": <asset_id>}`?

Then instead of doing:
```
Job.create(
    kwargs=dict(
        asset_or_sensor_id=asset_or_sensor.id,
        entity_type=asset_or_sensor.__class__.__name__,
    )
)
```
we could do:
```
Job.create(
    kwargs=dict(
        entity=asset_or_sensor,
   )
)
```

Reply:
I've created the functions `get_asset_or_sensor_ref` and `get_asset_or_sensor_from_ref` to get a reference (dictionary with class and ID) from an object and the other way around .